### ISSUE TITLE
No info about this app. | iOS Real Device

### ISSUE THREAD

**Issue by tvhimanshu** [kimxogus/react-native-version-check#92]
**No info about this app.**
@kimxogus 
I am getting this error in real iOS device. However its working fine in iOS Simulator and Android devices.

Suggestions appreciated.
Thanks

**Comment by kimxogus** [comment]
Sorry for late response.
As I don't have a real iOS device, it's hard for me to debug this issue

**Comment by nguyenvanphuc2203** [comment]
same issue

**Comment by jojonarte** [comment]
same issue :'(

**Comment by nguyenvanphuc2203** [comment]
@jojonarte i fixed it by use react-native-appstore-version-checker
check latest_version, source code like this
![Screen Shot 2019-11-22 at 4 24 41 PM](https://user-images.githubusercontent.com/33291479/69414125-b5f6a980-0d44-11ea-9837-bb7401230bfb.png)

**Comment by Saehanseul** [comment]
same

**Comment by ivanraybuglosa** [comment]
For outside US apps, please try to check your country id, I also have this error and I solved it this way.

`const storeSpecificId = Platform.OS == 'ios' ? "YOUR_APPSTORE_ID_HERE": "YOUR_GOOGLE_PLAYSTORE_ID.com_HERE";
    getAppstoreAppMetadata(storeSpecificId, { typeOfId: "id", country: "ph" })
      .then(appVersion => {
        console.log(appVersion); //latest version from appstore or playstore
      })
      .catch(error => {
        console.log("error occured", error);
      })`

**Comment by joony0123** [comment]
Hey guys!
If anyone is still struggling with this issue, I found out there is a problem with auto linking. I strongly recommend to try manual linking which solved for me

In addition while using Cocoa pod for iOS,
I found out my auto linking was adding below line to `tvOSTests` target not my main app target..
` pod 'react-native-version-check', :path => '../node_modules/react-native-version-check'`
When I added that line to my main target and ran `pod install` I was able to use it with my iOS device too

In conclusion, 
1. There seems to be a issue with autolinking
2. Try manual linking (if using Cocoa, add it to the right target and run `pod install`

Happy coding!

**Comment by rf1804** [comment]
Manual or `react-native link` has been depreceated, please follow new rules as per official react-native docs

- `yarn add react-native-version-check` or `npm install react-native-version-check`

- Only for ios: `pod install` or `pod update` as per requirement

**Comment by Hadajung** [comment]
If you have different bundle identifier, also you should check :) 
Under build settings > Packaging > Product Bundle Identifier.

**Comment by AnasAlamin** [comment]
i'm using 'react-native-version-check-expo' and this is also not working on ios real device... any workaround?

**Comment by AlphaJuliettOmega** [comment]
> Manual or `react-native link` has been depreceated, please follow new rules as per official react-native docs
> 
> * `yarn add react-native-version-check` or `npm install react-native-version-check`
> * Only for ios: `pod install` or `pod update` as per requirement

It seems your comment doesn't reference the issue, why is this ticket closed?

on iOs I have run Pod Install and Pod update many times, the `react-native-version-check`.getLatestVersion() worked for a few weeks but sometime in the last 2 weeks it just stopped working, with the error in the Ticket Title.

It would be nice to know why or how this error is caused.
